os/arch/Kconfig

```diff
@@ -454,6 +454,19 @@ config ARCH_IRQPRIO
 		interrupts with certain configuration options.  So this selection
 		should be used with caution.
 
+config ARCH_MMUDUMP
```

Comment:
Please check first commit, it's another commit and it has your changes. Something wrong.